### PR TITLE
show feeling history as a timeline in feelings page

### LINKED ISSUE #1692: Show feelings history on-screen
**Is your feature request related to a problem? Please describe.**
It's very annoying when you have to go through a lot of extra navigation clicks to see the history of your feelings.

**Describe the solution you'd like**
Show the history of your feelings on the Feelings page. Skip any dates without feeling. One date could have multiple feelings.

**Describe alternatives you've considered**
None.

**Additional context**

### PR DESCRIPTION
**Pull request by vinaybadgujar102**
resolves #1692

### CODE REVIEW

**Review by tijlleenders — CHANGES_REQUESTED** [review]
This is good - just reverse the order:

**Comment by vinaybadgujar102** [comment]
@tijlleenders check if the order is correct now.

**Review by tijlleenders — DISMISSED** [review]
The ordering is good now ... but somehow the Add button doesn't work after letting one day pass.

**Comment by vinaybadgujar102** [comment]
it seems to work correctly on my setup. do add button stops working for just the feelings page or overall app. can you post screen recording?

**Comment by tijlleenders** [comment]
The screen recording is not interesting - since there is nothing visibly going wrong. 
The Add button works correctly on the MyGoals page. It's only on MyFeelings that it doesn't work. If you can't reproduce then let's see after we merge.

**Comment by tijlleenders** [comment]
It works in Chrome.

**Review by tijlleenders — APPROVED** [review]

**Comment by tijlleenders** [comment]
@Tushar-4781 OK to merge?

File: src/pages/ShowFeelingsPage/useFeelingsData.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { useState, useEffect } from "react";
+import { getAllFeelings } from "@api/FeelingsAPI";
+import { getJustDate } from "@utils";
+import { feelingListType } from "@src/global";
+import { IFeelingItem } from "@src/models";
+
+const useFeelingsData = (trigger: boolean) => {
```
Comment (Tushar-4781) [inline src/pages/ShowFeelingsPage/useFeelingsData.ts:7]: Nice work @vinaybadgujar102 move this hook to the hooks folder

**Review by Tushar-4781 — CHANGES_REQUESTED** [review]

**Review by tijlleenders — APPROVED** [review]

**Pull request merged**